GDAXSharp.Specs/Services/Orders/OrdersServiceSpecs.cs

```diff
@@ -171,9 +172,10 @@ class when_cancelling_order_by_id_with_failure
             Because of = () =>
                 exception = Catch.Exception(() => Subject.CancelOrderByIdAsync("144c6f8e-713f-4682-8435-5280fbe8b2b4").Result);
 
-            It should_have_correct_error_response_message = () =>
+            private It should_have_correct_error_response_message = () =>
```

Comment:
remove extra private :) RESHARPER!!

Reply:
doh, every time :) yeah it automatically adds it